### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by rooc** [emmetio/livestyle-sublime-old#136]
Just tried to add stylesheet and got this

[15:30] Can’t diff: no previous state for 

Ubuntu, sublime 3
##

**Comment by sergeche** [comment]
Please try new alpha: http://livestyle.emmet.io/alpha/

**Comment by rooc** [comment]
I know, this is the version I'm talking about

http://screencloud.net/v/ktAM

**Comment by sergeche** [comment]
Can you post a screenshot with error log and opened LiveStyle popup?

**Comment by rooc** [comment]
http://screencloud.net/v/lZaQ
http://screencloud.net/v/a4dw

**Comment by sergeche** [comment]
Can you open `background.html` page as described here? http://livestyle.emmet.io/alpha/#error-reporting
And screenshot of log from that page

**Comment by rooc** [comment]
http://screencloud.net/v/3262

**Comment by rooc** [comment]
Don't know if it relevant but I have stylebot extension which can override site styles and in this case there are styles for that particular site which I try to style with Livestyle

**Comment by sergeche** [comment]
From that screenshot I see it works fine for stylesheet from typekit. And it seems like it doesn’t work for user stylesheets? I’ll check this use case later

**Comment by sergeche** [comment]
Can you try to reload page and see if other updates are applied to your custom stylesheet?

**Comment by rooc** [comment]
nope, same here

**Comment by sergeche** [comment]
Can you post stylesheet file you’re working with in editor?

**Comment by rooc** [comment]
https://www.dropbox.com/s/bxrjt05uwzlan00/pocket.css?dl=0

**Comment by sergeche** [comment]
Hmm, doesn’t look like a complex CSS, might be an issue with user stylesheet

**Comment by sergeche** [comment]
Please update both Chrome and Sublime Text plugins, this issue should be fixed

**Comment by rooc** [comment]
pulled plugin from repo
removed and installed again chrome extension from http://download.emmet.io/livestyle-alpha.crx

same story

**Comment by sergeche** [comment]
Can you post errors screenshots?

**Comment by rooc** [comment]
http://screencloud.net/v/xKm8
http://screencloud.net/v/124U
http://screencloud.net/v/yGpj

**Comment by sergeche** [comment]
Can you describe, step-by-step, what you did in browser and editor?

**Comment by rooc** [comment]
- wrote some styles in sublime without including anywhere
- opened chrome, went to livestyle dialog:  
   checked on, editor > browser, add stylesheet > picked pocket.css
- back to sublime - save
- error

**Comment by sergeche** [comment]
Just pushed update, do you still have this issue?

**Comment by rooc** [comment]
yep, the same

http://screencloud.net/v/iCuL
http://screencloud.net/v/AZ5W

**Comment by sergeche** [comment]
Can youb open ST console and check if there are errors?

**Comment by rooc** [comment]
http://screencloud.net/v/haGU

**Comment by sergeche** [comment]
No, everything seems fine.

Here’s what this problem means. You have an untitled file in ST which is treated by LiveStyle as CSS/LESS/SCSS (wrong syntax?). LiveStyle tries to save initial state of this file but it can’t be parsed (so initial state is not saved). And when you edit this untitled file, LS tries to diff current state against initial one and you receive this error.

Does it makes sense with your editor setup?

**Comment by rooc** [comment]
Not sure that I understood what you're saying. My file is readability.css, (/mnt/Chest/Dropbox/Data/StyleBot/readability.css) you can see it in the screen, so it is not untitled, syntax is usual CSS. Can this problem be somehow related to write access? Maybe my folder should have rights to wrtie files to everyone?

**Comment by sergeche** [comment]
Can you post a screenshot of your ST?
LiveStyle doesn’t use file system (it only reads preprocessor dependencies and never writes anything)

**Comment by rooc** [comment]
http://screencloud.net/v/uyYc

**Comment by sergeche** [comment]
Hmm, that’s very strange... I’ll check it on ST on Ubuntu

**Comment by rooc** [comment]
Yep, and when I worked with previous version of LiveStyle I did more or less the same procedure and it worked

**Comment by rooc** [comment]
I've tried it today at home and it works. There was the same error message but it started working
